requirements.in

```diff
@@ -13,13 +13,13 @@ click-datetime==0.2
 git+https://github.com/benoitc/gunicorn.git@1299ea9e967a61ae2edebe191082fd169b864c64#egg=gunicorn[eventlet]==20.1.0
 iso8601==1.0.2
 itsdangerous==2.1.2
-jsonschema==3.2.0
```

Comment:
Optional: As per https://github.com/python-jsonschema/jsonschema/blob/main/setup.cfg#L39, if we change this line to be 
`jsonschema[format]==4.4.0`
then we can remove the requirement on `rfc3339-validator` and `rfc3987` (assuming we don't need them ourselves)

Reply:
We do need both `rfc3339-validator` and `rfc3987` (I added `rfc3339-validator` in the second commit in this PR). I like the idea of not having to include them as top level dependencies and getting them automatically included with `jsonschema[format]` though so have added a new commit to make that change [9a249dc](https://github.com/alphagov/notifications-api/pull/3512/commits/9a249dc530487a6bf83062fb4460da0f1278f27d)